tools/hosting/docker-compose.yml

```diff
@@ -58,7 +58,7 @@ services:
 
   # FossilDB
   fossildb:
-    image: scalableminds/fossildb:master__442
+    image: scalableminds/fossildb:master__489
```

Comment:
Cc @normanrz -- this resolved https://forum.image.sc/t/ambiguity-for-setting-application-conf-for-serving-annotations/98112/5

thank you again!